lib/Wikia/src/Service/User/Permissions/PermissionsServiceImpl.php

```diff
@@ -0,0 +1,391 @@
+<?php
+
+namespace Wikia\Service\User\Permissions;
+
+class PermissionsServiceImpl implements PermissionsService {
+
+	/** @var  PermissionsConfiguration */
+	private $permissionsConfiguration;
+
+	/** @var string[string] - key is user id */
+	private $localExplicitUserGroups = [];
+
+	/** @var string[string] - key is the user id */
+	private $globalExplicitUserGroups = [];
+
+	/** @var string[string] - key is the user id */
+	private $implicitUserGroups = [];
+
+	/** @var string[string] - key is user id */
+	private $userPermissions = [];
+
+	public function __construct() {
+		$this->permissionsConfiguration = new PermissionsConfigurationImpl();
+	}
+
+	public function getConfiguration() {
+		return $this->permissionsConfiguration;
+	}
+
+	public function getExplicitGroups(\User $user ) {
```

Comment:
you could declare
`use User;`
just after the namespace statement at the top of the file and then you can use `User` instead of `\User` in this file.